.github/workflows/tests.yml

```
@@ -47,22 +47,22 @@ jobs:
 
       - name: Install UV
         uses: astral-sh/setup-uv@v2
         with:
           version: "latest"
 
       - name: Set up Python
-        uses: actions/setup-python@v5
+        uses: actions/setup-python@ea165f8d65b6e75b540449e92b4886f43607fa02
         with:
           cache: 'pip'
         with:
```

actionlint (1.7.12)

[error] key "with" is duplicated in element of "steps" section. previously defined at line:59,col:9

(syntax-check)